new/LIT7366Bakama.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7366Bakama" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer including ʾasmāt</title>
+                <title xml:lang="gez" xml:id="t2">በከመ፡ ረበቦሙ፡ ሰሎሞን፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ba-kama rababomu Salomon</title>
+                <title xml:lang="en" corresp="#t2">As Solomon scattered…</title>
```

Comment:
Perhaps a silly question:  why not to include ለአጋንንት፡ too in the title? So it seems a complete sentence.
The translation would sounds "As Solomon scattered the demons"

Reply:
I'm fine with either! :)